exec.go

```diff
@@ -703,12 +705,31 @@ func tab(et *Text, _ *Text, argt *Text, _, _ bool, arg string) {
 		args := strings.Split(arg, " ")
 		arg = args[0]
 		p := arg
+		if len(p) == 0 {
+			return
+		}
 		if '0' <= p[0] && p[0] <= '9' {
 			tab, _ = strconv.ParseInt(p, 10, 16)
 		}
 	}
 	if tab > 0 {
 		if w.body.tabstop != int(tab) {
+
+			// replace spaces with new tab width
+			if w.body.texpand {
```

Comment:
This code updates the existing text? Just using an Edit command would seem to be much more the Acme way?

Reply:
You're right, that would be more idiomatic. My thought process here was to replicate the behavior of executing `Tab <width>` when tab expansion is off by re-indenting automatically when tab expansion is activated. In any case, I've updated my PR to remove this unless you think there is a more elegant way of handling it.